.lintrunner.toml

```diff
@@ -62,7 +62,7 @@ init_command = [
     'python3',
     'tools/linter/adapters/pip_init.py',
     '--dry-run={{DRYRUN}}',
-    'clang-tidy==15.0.2.1',
+    'clang-tidy==14.0.6',
```

Comment:
clang-tidy 15.0.2 has issue linting this PR, clang-tidy 15.0.7 fixes that issue, but unfortunately PyPI does not have 15.0.7, so I change it to 14.0.6 instead.

Reply:
cc @jjsjann123